src/components/GenerateOfferButton.tsx

```diff
@@ -8,10 +8,10 @@ const GenerateOfferButton: React.FC<GenerateOfferButtonProps> = ({ disabled, onP
   return (
     <button
       onClick={onPressed}
-      className={`${disabled ? 'bg-brandDark/10 text-brandDark/20 dark:text-brandLight/30 cursor-not-allowed' : (isBuySelected ? 'bg-green-800' : 'bg-red-700')} text-brandLight px-4 py-2 rounded-lg w-full mt-8 font-medium`}
+      className={`${disabled ? 'bg-brandDark/10 text-brandDark/20 dark:text-brandLight/30 cursor-not-allowed' : (isBuySelected ? 'bg-green-800 hover:opacity-95' : 'bg-red-700 hover:opacity-95')} text-brandLight p-4 rounded-xl w-full mt-8 font-medium`}
       disabled={disabled}
     >
-      Generate Offer
+      Swap
```

Comment:
maybe have the text as an argument? wouldn't say 'Swap' for add/remove liquidity

Reply:
Update: did it myself